lib/vim_move.ahk

```diff
@@ -172,7 +172,8 @@
     }else if(key == "g"){
       Send, ^{Home}
     }else if(key == "+g"){
-      Send, ^{End}
+      ; Finish at start of last line, because that's what Vim does.
```

Comment:
This kind of comments should be written in the comment of the PR, not in the source code.

Reply:
I disagree. I think source comments should reflect the reason why code is written a certain way, and any reader coming across it in the future might be confused why we go back to the start of the first line.

Reply:
No, vim_ahk' concept is following Vim's behavior.
This is a kind of my miss implementation.

If you would like to add like this, we should write same comment everywhere...

Reply:
That makes sense. I'll remove it.